apps/smart-contracts/core/test/unit/DepositRecord.test.ts

```diff
@@ -113,6 +126,40 @@ describe('=> DepositRecord', () => {
       )
     })
 
+    it('correctly add to deposit totals and user totals for an uncapped user', async () => {
+      await depositRecord.setUserDepositCap(TEST_AMOUNT_ONE)
+      bypassList.isIncluded.whenCalledWith(uncappedUser.address).returns(true)
+      const globalDepositsBefore = await depositRecord.getGlobalNetDepositAmount()
+      const accountDepositsBefore = await depositRecord.getUserDepositAmount(uncappedUser.address)
```

Comment:
nit: match the existing variable naming
```suggestion
      const userDepositAmountBefore = await depositRecord.getUserDepositAmount(uncappedUser.address)
```

same for other test